packages/fury-adapter-swagger/lib/parser.js

```diff
@@ -366,6 +366,45 @@ class Parser {
           });
         }
 
+        if (this.swagger.info.contact) {
+          this.withPath('contact', () => {
+            const { Link } = this.namespace.elements;
+            const { name, url, email } = this.swagger.info.contact;
+
+            if (url) {
+              const link = new Link();
+              link.relation = 'contact';
+              link.href = url;
+
+              if (name) {
+                link.title = name.clone();
```

Comment:
```suggestion
                link.title = name;
```

Name here isn't an element so no need for `clone()`. It should be a `String` instance, the setter on `link.title` should wrap it in a `StringElement`